.travis.yml

```diff
@@ -40,6 +51,9 @@ matrix:
       env: TEST_SUITE=backend
 # command to run tests
 script:
+  # We unset GEM_PATH here as a hack to work around Travis CI having
+  # broken all ruby on their guests.  It's possible they fixed their
+  # bug and we can remove this.
   - unset GEM_PATH
   - ./tools/travis/$TEST_SUITE
```

Comment:
Reading this file to see how TEST_SUITE is used, I think I want `script` to be next to (right after) `install` -- they're logically quite similar.

Probably up at the top where `install` is now, since these are kind of the main event, though maybe `install` could move down here after where we set up TEST_SUITE.